irrep/__gvectors.py

```diff
@@ -62,11 +62,27 @@ def calc_gvectors(
     igp = np.zeros(3)
     igall = []
     Eg = []
+    memory = np.full(10, True)
     for N in range(nplanemax):
+        flag = True
         if N % 10 == 0:
             print(N, len(igall))
-        if len(igall) >= nplane / (2 if spinor else 1):
-            break
+        # if len(igall) >= nplane / (2 if spinor else 1):
+        #     break
+        if len(igall) >= nplane / 2:    # Only enters if vasp
+            if spinor:
+                break
+            else:      # Sure that not spinors?
```

Comment:
This can be done with a simple elif [ else + if np.all() = elif np.all() ]. I left it like this to make the code more clear. Which is your opinion? Do you prefer it with an elif?

Reply:
It's ok like this